### ISSUE TITLE
IEM interface in SMRT 

### ISSUE THREAD

**Issue by cryptomare** [smrt-model/smrt#19]
Hi all, 

I am working on the implementation of SMRT for wet snowpacks. I tried using the IEM interface for rough surface scattering in the SMRT model. While performing the sensitivity analysis, I came across different trends of backscatter as a function of liquid water content. When I used only SMRT (without adding a rough interface), then I see the decreasing trend of backscatter as the liquid water content increased. However, after adding the rough interface via IEM in the snowpack, I observed a sudden increase of backscatter from the one in SMRT. Moreover, as the liquid water content increases, the backscatter (SMRT + IEM) increases. I am confused in interpreting the behavior of backscatter as a function of liquid water content. In the SAR image, as I can see low backscatter values when there is an increased melt (or liquid water content), I would be grateful if you could let me know the increasing trend of modeled backscatter when the IEM interface is added in the SMRT. 

Thanks a lot for your time, 

Cheers, 
Shashwat

**Comment by cryptomare** [comment]
Hi Ghislain, 

Thanks a lot for such a nice and detailed explanation. Appreciate it. 

Okay, I understand now about the backscattering response when the surface is wet and rough. And indeed, it would be really interesting to investigate the regions which are both wet and rough. A small follow-up question: is the roughness a consequence of wetness, during surface melting period? Because of the increase in the permittivity which makes the surface look a bit rough to microwave sensor? Or is it simply solely because of increased wetness? Are these two quantities indirectly related?   

Thanks again for your time, 

Shashwat

**Comment by ghislainp** [comment]
Even if the surface roughness does not change during melt (rms height and corr length constant), the change in permittivity is so large when snow get wet that the backscattering can radically change according to the simulations.
However, you're right that snow melt may change the surface roughness after some time. To my knowledge, heavy rain is very efficient to increase roughness in mountains (e.g. https://www.skirando.net/nivologie/pluie1.png) but I'm not sure it would apply to melt and on a flat surface. Melt also leads to density increasing and associated setlling might smooth the surface.

**Comment by cryptomare** [comment]
Okay, I understand. In my case, I have seen that until moderate roughness (of rms height 0.5 cm), there is a decrease in the backscatter trend, however after 0.7 cm, to the more rougher surfaces (as you indicated correctly), the backscatter is increasing as the liquid water content increases, without any dip. Interesting. Also, during the time of acquisition, we saw wetty snow surface, however the temperature was below freezing. As the days before the acquisition, the temperature indeed was above freezing. So, in order to keep the specific details, I have used IBA instead of Derived IBA (wetsnow permittivity) because of temperature constraint. 

And indeed, high precipitation can lead to the roughness changes of the surface, but it is interesting to note how these parameters can influence the others rising to a complicated backscattering response. 

Thanks again for your prompt responses. Appreciate it very much.
